pages/dkp/kommander/2.0/gpu/index.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: layout.pug
+navigationTitle: GPUs
+title: GPUs
+menuWeight: 150
+excerpt: Configure GPU for Kommander cluster
+beta: false
+enterprise: false
+---
+
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 -->
+
+This section describes the Nvidia GPU support on Kommander. This document assumes familiarity with Kubernetes GPU support. AMD GPUs are currently not supported.
+
+# Kommander GPU Overview
+
+GPU support on Kommander is achieved by using the [nvidia-container-runtime][nvidia_container_runtime].
+With the Nvidia GPU turned on, Kommander configures the container runtime for running GPU containers, and installs all the necessary items to power up the Nvidia GPU devices.
+Kommander runs every Nvidia GPU dependent component as daemonsets, making it easier to manage and upgrade.
```

Comment:
Perhaps move this sentence below the list of components. That would also make it more natural to say `Kommander runs these components as daemonsets,...`